README.md

```diff
@@ -1,5 +1,20 @@
 # calens
 
-This program is a changelog generator that nicely prints a changelog file from
+`calens` is a changelog generator that nicely prints a changelog file from
 a directory with versioned subdirs containing files. A sample directory can be
 seen in [the restic repository](https://github.com/restic/restic/tree/master/changelog)
+
+# Installation
+
+To install and run calens locally, clone this repo and run from the command line `go install`.
+The installation process downloads all required libraries. With some OS, you may need to install
+as prerequisite, if the OS provides it like Ubuntu, the `build essentials` package.
+The installation process responds in case which part is missing. 
+
+# Testing
+
+To test calens and create a `test-changlog.md` file you can open with a browser, change into
+a cloned repo satisfying the above requirements and run `calens -o test-changlog.md`.
```

Comment:
```
To test calens and create a `changelog.md` file, change into a repository containing
a `changelog` folder similar to that of [restic](https://github.com/restic/restic/tree/master/changelog).
Then run `calens -o changelog.md` from within the repository.
```

Reply:
Done with one exception. I did not change the filename to `changelog.md`, because if one already has calens running but want to see changes on the output, we should not overwrite or confuse with an existing `changelog.md` file.

Reply:
Please note that I have also filed a PR in https://github.com/restic/restic/pull/4567 to update the template. This update is necessary to make this change working as the current template does not use correct referencable markdown section headings. Also note that the change made is rather small, but looks big as I included an updated build changlog file.

Reply:
> Done with one exception. I did not change the filename to `changelog.md`, because if one already has calens running

I don't agree. If a user wants to diff against some existing changelog, then it's their task to adjust the CLI options accordingly.

There's already a calens 0.4.0 release with the recent changes.